com.avaloq.tools.ddk.xtext.ui.test/src/com/avaloq/tools/ddk/xtext/ui/quickfix/WorkbenchResolutionAdaptorTest.java

```diff
@@ -21,12 +21,13 @@
 import org.eclipse.xtext.ui.editor.quickfix.IssueResolution;
 import org.eclipse.xtext.ui.util.IssueUtil;
 import org.eclipse.xtext.validation.Issue;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 import com.avaloq.tools.ddk.xtext.ui.quickfix.WorkbenchMarkerResolutionGenerator.WorkbenchResolutionAdapter;
 
 
+@SuppressWarnings({"PMD.JUnit5TestShouldBePackagePrivate", "PMD.SignatureDeclareThrowsException"})
 public class WorkbenchResolutionAdaptorTest {

```

Comment:
This doesn't look right--isn't the proper fix here to remove the public modifier on the class? See https://stackoverflow.com/a/55230350/4700312
```suggestion
class WorkbenchResolutionAdaptorTest {
```